.github/workflows/create-prs-to-update-vcs-repositories.yaml

```
@@ -0,0 +1,30 @@
+name: create-prs-to-update-vcs-repositories
+
+on:
+  schedule:
+    - cron: 0 0 * * *
```

I think this workflow can complete within few minutes.
So that why don't you run 4 times per day?

```suggestion
    - cron: '0 0,6,12,18 * * *'
```